src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -26,6 +26,36 @@
 
 namespace Microsoft.ML.Transforms
 {
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="NormalizingTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | <xref:System.Single> or <xref:System.Double> |
```

Comment:
>  [](start = 79, length = 1)

I think numeric vectors are also valid, aren't they? If so please change here and also the params. you can write it like this:

<xref:System.Single>, <xref:System.Double>, or a known-sized vector of those types.

Reply:
Its only supports single or double:
[https://github.com/dotnet/machinelearning/blob/ea5c095f90cced3bd2c67da7289fe1be532f9ba4/src/Microsoft.ML.Data/Transforms/Normalizer.cs#L271](https://github.com/dotnet/machinelearning/blob/ea5c095f90cced3bd2c67da7289fe1be532f9ba4/src/Microsoft.ML.Data/Transforms/Normalizer.cs#L271)


---
In reply to: [276994571](https://github.com/dotnet/machinelearning/pull/3432#discussion_r276994571) [](ancestors = 276994571)

Reply:
i think known-sized vectors of numerics are supported by normalizer. that line you're referencing is also true if the vector contains only numeric values. could you please confirm with Ivan?

I'm certain that I've normalized the feature **vector** right before trainer

---
In reply to: [277045796](https://github.com/dotnet/machinelearning/pull/3432#discussion_r277045796) [](ancestors = 277045796,276994571)

Reply:
Look on line 298 in this file.
It's either scalar or vector of known-size of float or double

---
In reply to: [277069180](https://github.com/dotnet/machinelearning/pull/3432#discussion_r277069180) [](ancestors = 277069180,277045796,276994571)

Reply:
sorry I read too quickly through Shahab's original comment and thought he was asking if we supported integers. Yes we do support vectors, sorry for the confusion. I am updating.

---
In reply to: [277070189](https://github.com/dotnet/machinelearning/pull/3432#discussion_r277070189) [](ancestors = 277070189,277069180,277045796,276994571)